src/screens/dashboard.tsx

```diff
@@ -48,6 +46,17 @@ const DashboardScreen = (): JSX.Element => {
   console.log('data', auditProgram)
   console.log('survey', survey)
 
+  const arrayOfTaskArray = Object.values((survey as any).tasks)
+
```

Comment:
I just realized you are usingh this same logic in the ```tasks``` screen. Lets keep the code dry, create a method an put it in the ```utils``` folder